src/commands/createFunctionApp/stacks/getStackPicks.ts

```diff
@@ -176,6 +178,46 @@ async function getStacks(context: IFunctionAppWizardContext & { _stacks?: Functi
     return context._stacks;
 }
 
+async function getFlexStacks(context: IFunctionAppWizardContext & { _stacks?: FunctionAppStack[] }): Promise<FunctionAppStack[]> {
+    const client: ServiceClient = await createGenericClient(context, context);
+    const location = await LocationListStep.getLocation(context);
+    const flexFunctionAppStacks: FunctionAppStack[] = [];
+    const stacks = ['dotnet', 'java', 'node', 'powershell', 'python'];
+    if (!context._stacks) {
+        const getFlexStack = async (stack: string) => {
+            const result: AzExtPipelineResponse = await client.sendRequest(createPipelineRequest({
+                method: 'GET',
+                url: requestUtils.createRequestUrl(`providers/Microsoft.Web/locations/${location.name}/functionAppStacks`, {
```

Comment:
so flex stacks are location dependent? That's crazy

Reply:
Yeah, I can see it being helpful for this team since they have to deploy skus to regions individually (it seems like).